lib/jekyll-redirect-from/redirector.rb

```diff
@@ -82,5 +82,9 @@ def config_github_url(site)
     def config_baseurl(site)
       site.config.fetch('baseurl', nil)
     end
+
+    def config_url(site)
+      site.config.fetch('url', nil)
```

Comment:
@benbalter  `URI.join` seems to fail for relative urls. What about `File.join` as used within the same class?

```ruby
def redirect_prefix(site)
  config_github_url(site) || config_url(site)
end

def config_url(site)
  url = site.config.fetch('url', nil) || ""
  baseurl = site.config.fetch('baseurl', nil) || ""
  File.join url, baseurl
end
```